apollo-ios/Sources/Apollo/GraphQLResult.swift

```diff
@@ -47,3 +47,28 @@ extension GraphQLResult: Equatable where Data: Equatable {
 }
 
 extension GraphQLResult: Hashable where Data: Hashable {}
+
+extension GraphQLResult {
+  public func asJSONDictionary() -> [String: Any] {
```

Comment:
Just missing documentation for this new function and then I think it's good to go.